apps/discord-bot/src/commands/bedwars/bedwars.profile.tsx

```diff
@@ -112,3 +113,51 @@ export const BedWarsProfile = ({
     </Container>
   );
 };
+
+export const BedWarsChallenges = ({
```

Comment:
Most to its own file, `bedwars-challenges.profile.tsx`, and rename to `BedWarsChallengesProfile`